AdChoicesManager.js

```diff
@@ -2,47 +2,38 @@ import React, {PropTypes} from 'react';
 import {requireNativeComponent, StyleSheet, Platform} from 'react-native';
 
 const NativeAdChoicesView = requireNativeComponent('AdChoicesView', null);
-let positionStyle = null;
 
 type adChoiceLocation = 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight';
 
 export default class AdChoicesView extends React.Component<Object> {
```

Comment:
@Suraj-Tiwari should we use the same PR for addressing this component's prop types to something more specific than `Object`?

Reply:
@erandagan It would be cool if you add more specific Prop and State types.

Reply:
@Trancever approve that revert pull request and merge this when fixed. I'll be back once i'm done sleeping.